packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -42,6 +42,7 @@ type SelectControlProps< ItemType > = {
 	onSelect?: ( selected: ItemType ) => void;
 	placeholder?: string;
 	selected: ItemType | ItemType[] | null;
+	dropdownPlacement?: 'inline' | 'body';
```

Comment:
Using `body` will depend on the `Popover.Slot` being used, maybe I should add a warning if that is not available?
If `Popover.Slot` is not used it will just default to rendering it `inline`.